packages/x-date-pickers/src/MonthCalendar/PickersMonth.tsx

```diff
@@ -55,7 +59,7 @@ const PickersMonthRoot = styled('div', {
   variants: [{ props: { monthsPerRow: 4 }, style: { flexBasis: '25%' } }],
 });
 
-const PickersMonthButton = styled('button', {
+export const PickersMonthButton = styled('button', {
```

Comment:
Do you think we should rename this component now or in v8?
For now it's private so renaming it is not a breaking change, but its name would not be coherent with its theme entry.

If in v8 this component is directly called inside `MonthCalendar`, it could be named `MonthCalendarButton` or `MonthCalendarElementButton`, or `MonthCalendarMonthButon` but it's super ugly.

Reply:
I would be very much for renaming now, just to avoid the unnecessary change in v8, but indeed, the naming confusion between the exported component and the theme entry doesn't seem elegant. 🙈 
On another note—do we really need to export it?
I would imagine users wanting to pass their own components instead of styling this one as it's a considerably small/simple component. 🤔

Reply:
> On another note—do we really need to export it?

I think we always export the slots so that people can easily create wrappers.
This component is probably not the one for which the export is the most valuable.

Reply:
> If we rename it, which name would you pick?

I'd vote for `MonthCalendarButton`, but in this case, I'd vote for not exporting, at least until there is demand for it. 🤔

Reply:
Done :+1: